autodoc-admin-api/data.lua

```diff
@@ -637,6 +737,26 @@ return {
           * `all` means "run on all nodes", meaning both sidecars in a sidecar-to-sidecar
             scenario. This is useful for tracing/logging plugins.
         ]] },
+        protocols = {
+          description = [[
+            A list of the request protocols that will trigger this plugin. The default value is
+            `{ "http", "https" }`. Possible values for protocols are `"http"`, `"https"`, `"tcp"`, and `"tls"`.
+            The default value and available values for this field might change depending on the plugin type.
```

Comment:
this sentence, while true, can confuse users. Maybe adding a clarification would help:

```
For example, plugins that only work in stream mode will only support `"tcp"` and `"tls"`.
```